sympy/series/fourier.py

```diff
@@ -96,19 +96,17 @@ def finite_check(f, x, L):
     def check_fx(exprs, x):
         return x not in exprs.free_symbols
 
-    def check_sincos(expr, x, L):
-        if type(expr) == sin or type(expr) == cos:
-            sincos_args = expr.args[0]
+    def check_sincos(_expr, x, L):
+        if type(_expr) == sin or type(_expr) == cos:
```

Comment:
This could probably be `if isinstance(_expr, (sin, cos))`.

Reply:
I can update it.